apps/gdalinfo_lib.cpp

```diff
@@ -215,6 +216,174 @@ gdal_json_object_new_double_significant_digits(double dfVal,
+        .help(_("Display the output in json format."));
+
+    argParser->add_argument("-mm")
+        .store_into(psOptions->bComputeMinMax)
+        .help(_("Force computation of the actual min/max values for each band "
+                "in the dataset."));
+
+    {
+        auto &group = argParser->add_mutually_exclusive_group();
+        group.add_argument("-stats")
+            .store_into(psOptions->bStats)
+            .help(_("Read and display image statistics computing exact values "
+                    "if required."));
+
+        group.add_argument("-approx_stats")
+            .store_into(psOptions->bApproxStats)
+            .help(
+                _("Read and display image statistics computing approximated "
+                  "values on overviews or a subset of all tiles if required."));
+    }
+
+    argParser->add_argument("-hist")
+        .store_into(psOptions->bReportHistograms)
+        .help(_("Report histogram information for all bands."));
+
+    // Helper to set an inverted logic flag
+    auto setInvFlag = [&argParser](const std::string &name, bool &storage,
+                                   const std::string &help)
+    {
+        argParser->add_argument(name)
+            .default_value(true)
+            .implicit_value(false)
+            .action([&storage](const auto &) { storage = false; })
+            .help(help);
+    };
+
+    setInvFlag("-nogcp", psOptions->bShowGCPs,
+               _("Suppress ground control points list printing."));
+
+    setInvFlag("-nomd", psOptions->bShowMetadata,
+               _("Suppress metadata printing."));
+
+    setInvFlag("-norat", psOptions->bShowRAT,
+               _("Suppress printing of raster attribute table."));
+
+    setInvFlag("-noct", psOptions->bShowColorTable,
+               _("Suppress printing of color table."));
+
+    setInvFlag("-nofl", psOptions->bShowFileList,
+               _("Suppress display of the file list."));
+
+    argParser->add_argument("-checksum")
+        .flag()
+        .store_into(psOptions->bComputeChecksum)
+        .help(_(
+            "Force computation of the checksum for each band in the dataset."));
+
+    argParser->add_argument("-listmdd")
+        .flag()
+        .store_into(psOptions->bListMDD)
+        .help(_("List all metadata domains available for the dataset."));
+
+    argParser->add_argument("-proj4")
+        .flag()
+        .store_into(psOptions->bReportProj4)
+        .help(_("Report a PROJ.4 string corresponding to the file's coordinate "
+                "system."));
+
+    argParser->add_argument("-wkt_format")
+        .metavar("<WKT1|WKT2|WKT2_2015|WKT2_2018>")
+        .choices("WKT1", "WKT2", "WKT2_2015", "WKT2_2018")
+        .nargs(1)
+        .store_into(psOptions->osWKTFormat)
+        .help(_("WKT format used for SRS."));
+
+    argParser->add_argument("-sd")
+        .metavar("<sd>")
+        .nargs(1)
+        .store_into(psOptionsForBinary->nSubdataset)
+        .help(
```

Comment:
In https://github.com/OSGeo/gdal/pull/9683/files, I use this shorter help message:
```
    argParser->add_argument("-sd")
        .metavar("<n>")
        .store_into(psOptions->nSubdataset)
        .help(_("Use subdataset of specified index (starting at 1), instead of "
                "the source dataset itself."));
```

Reply:
done